src/editorapp.cc

```diff
@@ -654,6 +692,18 @@ void EditorApp::on_actionMusicFile_triggered()
 	}
 }
 
+void EditorApp::on_actionVideoFile_triggered()
+{
+	QString fileName = QFileDialog::getOpenFileName(this, tr("Open File"),
+			latestPath,
+			tr("Video files") + " (*.mp4 *.avi)");
```

Comment:
Thanks for your contribution 👍 
Looking good so far. Still need to test it

Within performous we can read the following extensions (splitted per `\`)
`avi|mpg|mpeg|flv|mov|mp4|mkv|m4v`
So maybe add those aswell

Reply:
Ok, thanks for the feedback. I pushed the changes and hope I have done it right.